### ISSUE TITLE
sort in list utilities library very slow

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap#2440]
Sorting 1000 numbers took almost 8 seconds on my laptop. Project attached.

![image](https://user-images.githubusercontent.com/2138696/60385027-9f417480-9a7c-11e9-99ae-8a224ca6781c.png)

[sort timer.zip](https://github.com/jmoenig/Snap/files/3341929/sort.timer.zip)

**Comment by jmoenig** [comment]
Yeah, better to use the JIT-compiled version in the *bigger data* or *frequency distribution analysis* libraries.

**Comment by bromagosa** [comment]
I can confirm using the `bigger data` sort is almost instant. It's building the actual list that takes most of the time (warping the repeat block fixes that too).

**Comment by brianharvey** [comment]
I've been poking at this.  One issue is that using a plain build-your-own block is a lot faster than CALLing a lambda that you've put in a script variable because we don't have inner definitions.

**Comment by jmoenig** [comment]
@brianharvey as an old Smalltalker I wouldn't take any issue with a separate helper block in the palette, especially if it makes SORT faster...
